jena-arq/src/main/java/org/apache/jena/riot/lang/LangJSONLD11.java

```diff
@@ -45,13 +46,17 @@
 import org.apache.jena.riot.system.ParserProfile;
 import org.apache.jena.riot.system.StreamRDF;
 import org.apache.jena.sparql.util.Context;
+import org.apache.jena.sparql.util.Symbol;
 
 /**
  * JSON-LD 1.1 {@link ReaderRIOT}.
  */
 public class LangJSONLD11 implements ReaderRIOT {
     private final ErrorHandler errorHandler;
     private final ParserProfile profile;
+    
+    /** value: the option object expected by JsonLdProcessor (instance of JsonLdOptions) */
+    public static final Symbol JSONLD_OPTIONS = createSymbol("JSONLD_OPTIONS");
```

Comment:
Suggestion: 

`public static final Symbol JSONLD_OPTIONS = SystemARQ.allocSymbol(SYMBOLS_NS, "options");`

No need to say "jsonld" twice in the URI. `SystemARQ.allocSymbol` will do the symbol work. 

Names tend to be lowercase.

Reply:
Copied from LangJSONLD10. Made requested changes to that class as well.